js/anims-missiles.js

```diff
@@ -89,6 +112,20 @@ function listenKeypressMissiles() {
     })
 }
 
-createMissileEventHandler()
+/**
+ * Increases animation speed based upon gameScore.score
+ * We call this function inside repeatAnim function
+ */
+function animationChangeSpeed() {
+    var animationDuration = 3000
+    if (gameScore.score > 0) {
+        animationDuration = 3000 / (1 + (gameScore.score * 0.03))
+    } else {
+        animationDuration = 3000
+    }
+    return animationDuration
```

Comment:
if you return line 122, you can return 3000 here

Reply:
Switched with ternary operator in 0bfeea3